OWNERS_ALIASES

```diff
@@ -8,6 +8,10 @@ aliases:
     - onlydole
     - sftim
     - nate-double-u
+  sig-docs-bn-owners: # Admins for Bengali content
+    - mitul3737
+  sig-docs-bn-reviews: # PR reviews for Bengali content
+    - mitul3737
```

Comment:
We need minimum 2 owners to launch a localization.

Reply:
Let's hold this PR until there are 2 bn owners
/hold

Reply:
@mitul3737 See above – it would be great to have a second owner, that is a member of the Kubernetes org, listed here to officially kick off the localization

Reply:
Yes sure. Our another member is applying for being an org member. Till now we don't have more than 1 org member.  I think we should wait till her membership 😅